cloud_governance/common/clouds/aws/price/resources_pricing.py

```diff
@@ -0,0 +1,41 @@
+import os
+
+
+from cloud_governance.common.clouds.aws.price.price import AWSPrice
+
+
+class ResourcesPricing:
+
+    def __init__(self):
+        self._aws_pricing = AWSPrice()
+        self.region = os.environ.get('AWS_DEFAULT_REGION', 'us-east-1')
+
+    def ec2_instance_type_cost(self, instance_type: str, hours: float):
+        """
+        This method returns the cost of ec2 instance types cost
+        @return:
+        """
+        cost = float(self._aws_pricing.get_price(instance=instance_type, os='Linux', region=self._aws_pricing.get_region_name(self.region)))
+        return cost * hours
+
+    def get_ebs_cost(self, volume_size: int, volume_type: str, hours: float):
+        """
+        This method returns the cost of ebs_volume
+        @param hours:
+        @param volume_size:
+        @param volume_type:
+        @return:
+        """
+        cost = float(self._aws_pricing.get_ebs_cost(volume_type=volume_type, region=self.region))
+        return cost * volume_size * (hours / 730)
```

Comment:
Why dud you divide it to 730 ?

Reply:
I can't find a method for it. When I check prices it is const for all regions.

Reply:
I think that AWS charge up a monthly, also if you cancel it in a middle
Can you please check it ?

Reply:
Yes,they cost us how many hours we use.

Reply:
So create constant MONTHLY_HOURS=730